src/main/java/com/playtika/automation/school/test/framework/configuration/NotesConfiguration.java

```diff
@@ -0,0 +1,18 @@
+package com.playtika.automation.school.test.framework.configuration;
+
+import com.playtika.automation.school.test.framework.action.NotesActions;
+import com.playtika.automation.school.test.framework.client.NotesFeignClient;
+import org.springframework.cloud.openfeign.EnableFeignClients;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableFeignClients(clients = NotesFeignClient.class)
+public class NotesConfiguration {
+
+    @Bean
+    public NotesActions notesActions(NotesFeignClient notesFeignClient) {
+        return new NotesActions(notesFeignClient);
+    }
+
```

Comment:
This line is redundant. Some of your configuration classes have empty line, some classes do not have empty line. you should use the one code style